cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -84,7 +84,12 @@ type Etcd struct {
 type NodeConfiguration struct {
 	metav1.TypeMeta
 
-	Discovery Discovery
+	Discovery         Discovery
```

Comment:
Why do we have this and the separate fields below?  Are we going to remove/deprecate the `--discovery` flag in a separate PR?

If this is getting removed perhaps mark it as such?